### PR TITLE
feat: account info page

### PR DESCRIPTION
**Pull request by annechung**
## Notion ticket link

[My Account Screen](https://www.notion.so/uwblueprintexecs/Team-Hub-5b0b225bfc09415ebcfad3030ce5fd30?p=fb252d0f22a347348ceae963feb6392e)

## Implementation description
* account information for donor (view only) 
* a bit different from the designs, went over changes with binal!

## Steps to test
1. log in 
2. navigate to my account page from nav bar
3. verify the correct information is shown

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/Account.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    }
+    const getDonor = async () => {
+      const donor = await DonorAPIClient.getDonorByUserId(authenticatedUser.id);
+      setBusinessName(donor.businessName);
+    };
+    getDonor();
+  }, [authenticatedUser]);
+
+  if (!authenticatedUser) {
+    return <></>;
```
Comment (hanlinc27) [inline frontend/src/components/pages/Account.tsx:40]:
```suggestion
    return <Redirect to={LOGIN_PAGE}/>;
```

File: frontend/src/components/pages/Account.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              variant="unstyled"
+              size="sm"
+            />
+          </Box>
+          <Box mt="1rem">
+            <Text>Email address</Text>
+            <Input
+              mt="2"
+              name={authenticatedUser!.email}
+              value="email"
```
Comment (hanlinc27) [inline frontend/src/components/pages/Account.tsx:113]:
```suggestion
              name="email"
              value={authenticatedUser!.email}
```

File: frontend/src/components/pages/Account.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+              <Input
+                mt="2"
+                value={authenticatedUser!.firstName}
+                name="firstName"
+                placeholder="Enter first name"
+                variant="unstyled"
+                size="sm"
+              />
+            </Box>
+            <Box mt="1rem">
```
Comment (hanlinc27) [inline frontend/src/components/pages/Account.tsx:84]:
```suggestion
            <Box mt="1.5rem">
```
can we make this a bit larger?

**Review by hanlinc27 — CHANGES_REQUESTED** [review]
think the email field got flipped but looks good otherwise!! :)

File: frontend/src/components/pages/Account.tsx (hunk 2 above)
Comment (annechung) [reply frontend/src/components/pages/Account.tsx:113]: ooops good catch!

File: frontend/src/components/pages/Account.tsx (hunk 3 above)
Comment (annechung) [reply frontend/src/components/pages/Account.tsx]: fixed!

File: frontend/src/components/pages/Account.tsx (hunk 1 above)
Comment (annechung) [reply frontend/src/components/pages/Account.tsx]: added! thanks

**Review by hanlinc27 — APPROVED** [review]

File: frontend/src/components/common/Header.tsx [hunk 4]
```diff
@@ -75,7 +77,7 @@ const Header = (): JSX.Element => {
               <Link as={ReactLink} to={DASHBOARD_PAGE}>
                 Scheduled Donations
               </Link>
-              <Link as={ReactLink} to={LANDING_PAGE}>
+              <Link as={ReactLink} to={ACCOUNT_PAGE}>
                 {/* update link to account page */}
```
Comment (Shehryar21) [inline frontend/src/components/common/Header.tsx:81]: We can remove the comment.

**Review by Shehryar21 — APPROVED** [review]

**Pull request merged**